object-filter/src/main/java/org/infinispan/objectfilter/impl/syntax/parser/QueryRendererDelegateImpl.java

```diff
@@ -355,10 +355,11 @@ public void predicateFullTextTerm(String term, String fuzzyFlop) {
       if (property.isEmpty()) {
          throw log.getPredicatesOnEntityAliasNotAllowedException(propertyPath.asStringPath());
       }
-      String comparisonValue = (String) parameterValue(term);
+      checkAnalyzed(property,true);
```

Comment:
There's ```checkAnalyzed(property,true);``` inside the if statement already. Why do we need it here too?

Reply:
Where?

Reply:
Ok, saw it